summary and Q3 CSS

iravella opened the pull request.

iravella committed:
summary and Q3 CSS

emeliese commented on src/components/QuestionThree.css:
```diff
@@ -0,0 +1,33 @@
+  }
+  
+  /* Media query for smaller screens (adjust as needed) */
+  @media screen and (max-width: 600px) {
```
Is there a reason you didn't use the same width as the rest of the app? The drop down still extends outside of the frame when in mobile mode (I used google chromes dev tools and changed mode)

iravella replied:
Deleted the CSS code. The dropdown issue is currently pending a fix and will be addressed in the upcoming pull request (PR).

emeliese commented:
Don't forget to merge the latest changes from main into this branch :)

iravella committed:
Update QuestionThree.css

Deleted  @media screen and (max-width: 600px) and merged the commit. Pending fix.
